.github/workflows/release-published.yml

```diff
@@ -0,0 +1,22 @@
+name: Checks after release is published
+on:
+  release:
+    types: ['published']
```

Comment:
Do we only want to run this when a release is published? What if we have to pull a release because of a major breaking change? I'd still want to check that we're pulling the current functional one.

I'm not sure what all the release types mean – what's the difference between `published`, `created, and `released`?